_includes/v23.1/zone-configs/variables.md

```diff
@@ -1,8 +1,8 @@
+`range_max_bytes` | <a name="range-max-bytes"></a> The maximum size, in bytes, for a [range]({{link_prefix}}architecture/glossary.html#architecture-range) of data in the zone. When a range reaches this size, CockroachDB will [split it]({{link_prefix}}architecture/distribution-layer.html#range-splits) into two ranges.<br><br>**Default:** `536870912` (512 MiB)
+`gc.ttlseconds` | <a name="gc-ttlseconds"></a> The number of seconds overwritten [MVCC values]({{link_prefix}}architecture/storage-layer.html#mvcc) will be retained before [garbage collection]({{link_prefix}}architecture/storage-layer.html#garbage-collection). Smaller values can save disk space if values are frequently overwritten; larger values increase the interval allowed for [`AS OF SYSTEM TIME`](as-of-system-time.html) queries, also known as [Time Travel Queries](select-clause.html#select-historical-data-time-travel).<br><br>It is not recommended to set this below `600` (10 minutes); doing so will cause problems for [long-running queries](manage-long-running-queries.html). Also, since all versions of a row are stored in a single range that never splits, it is not recommended to set this so high that all the changes to a row in that time period could add up to more than [the default range size](#range-max-bytes); such oversized ranges could contribute to the server [running out of memory](cluster-setup-troubleshooting.html#out-of-memory-oom-crash) or other problems. {{site.data.alerts.callout_info}} Ensure that you set `gc.ttlseconds` long enough to accommodate your [backup schedule](create-schedule-for-backup.html), otherwise your incremental backups will fail with [the error message `protected ts verification error`](common-errors.html#protected-ts-verification-error). For example, if you set up your backup schedule to recur at an interval greater than `gc.ttlseconds`, all your incremental backups will fail.{{site.data.alerts.end}} **Default:** `14400` (4 hours)
```

Comment:
Re: "Ensure that you set `gc.ttlseconds` long enough to accommodate your..." this is now less applicable. GC TTL now has nothing to do with _scheduled_ backups since scheduled backups with revision history will automatically prevent garbage collection of relevant data regardless of gc.ttlseconds. If the scheduled backups don't use revision history, whether we GC overwritten data or not is irrelevant, since it does not appear in the backup.

The whole "keep your GC TTL large enough to accommodate your backups" only applies if you're not using scheduled backups, and are effectively issuing new backup statements yourself every N hours. If you're doing that, and also want revision history, then you do need to keep your GC TTL large enough. But our guidance is to prefer using scheduled backups.

Do we have other references to this "coupling" between backups and the GC TTL?

Reply:
This link might be helpful here, re coupling of non-scheduled backups to GC TTL https://www.cockroachlabs.com/docs/v22.2/create-schedule-for-backup#protected-timestamps-and-scheduled-backups

Reply:
> Re: "Ensure that you set `gc.ttlseconds` long enough to accommodate your..." this is now less applicable. GC TTL now has nothing to do with _scheduled_ backups since scheduled backups with revision history will automatically prevent garbage collection of relevant data regardless of gc.ttlseconds. If the scheduled backups don't use revision history, whether we GC overwritten data or not is irrelevant, since it does not appear in the backup.

> The whole "keep your GC TTL large enough to accommodate your backups" only applies if you're not using scheduled backups, and are effectively issuing new backup statements yourself every N hours. If you're doing that, and also want revision history, then you do need to keep your GC TTL large enough. But our guidance is to prefer using scheduled backups.

> Do we have other references to this "coupling" between backups and the GC TTL?



> This link might be helpful here, re coupling of non-scheduled backups to GC TTL https://www.cockroachlabs.com/docs/v22.2/create-schedule-for-backup#protected-timestamps-and-scheduled-backups

Awesome, thanks Kathryn - I've linked to this in the revised text, PTAL

Reply:
This part is not correct:

```
For example, if you set up your backup schedule to recur at an interval greater than `gc.ttlseconds`, all your incremental backups will fail.
```

What instead happens now is that if you're using scheduled backups with revision history + an interval greater than the GC TTL, we'll effectively only GC data once it's been backed up, independent of the GC TTL. These backups "chain together" because we know the full schedule, and prevent GC of revisiones you wanted to have backed up. So backups will not fail. They used in in some releases older than 22.2, the chaining was enabled by default in 22.2: https://github.com/cockroachdb/cockroach/pull/79976.

Reply:
I have removed that sentence altogether in the latest update (to both v22.2 and v23.1 docs).  That section now reads:

> If you are not yet using [scheduled backups](create-schedule-for-backup.html), and instead issuing [`BACKUP`](backup.html) statements manually, you must ensure that you set `gc.ttlseconds` long enough to accommodate your manual backup schedule. Otherwise, your incremental backups will fail with [the error message `protected ts verification error`](common-errors.html#protected-ts-verification-error). We recommend using [scheduled backups](create-schedule-for-backup.html) instead, which automatically [use protected timestamps](create-schedule-for-backup.html#protected-timestamps-and-scheduled-backups) to ensure they succeed.

what do you think?

Reply:
That looks great!